tests/unit/Parameters/EndMeetingParametersTest.php

```diff
@@ -36,4 +36,27 @@ public function testEndMeetingParameters()
         $this->assertEquals($newId, $endMeetingParams->getMeetingID());
         $this->assertEquals($newPassword, $endMeetingParams->getPassword());
     }
+
+    /**
+     * @group legacy
+     */
+    public function testEndMeetingParametersWithoutPassword(): void
```

Comment:
@FelixJacobi Isn't this something that is checking the opposite. We used to require a password on the EndMeetingParameter constructur. We now deprecated this by allowing a null password. This test should therefore check if an exeception is thrown if you pass a password and not if you dont.

Reply:
No, the test tests if we're throwing an exception when you try to get a password, that does not exist anymore.

Allow to return null was not possible, since that would be a breaking change of the class interface.

Reply:
The upgrade path on next major is later, that this test and the getPassword method is completely removed (again). Throwing an exception is just a transitional workaround.